examples/synergy_create_server_profile_with_deployment_plan.yml

```diff
@@ -16,15 +16,16 @@
 ---
 ###
 # This playbook is only compatible with HPE Synergy Enclosures
+# This playbook will create server profile with Nic connections using server profile template, deployment plan and os clustom attributes
```

Comment:
Type error: it should be os custom attributes

Reply:
done